docs/content/commands/npm-diff.md

```diff
@@ -155,7 +155,7 @@ located within the folder `./lib/` and changed lines of code within the
 
 #### diff
 
-* Type: Array<String>
+* Type: Array&lt;String&gt;
```

Comment:
```suggestion
* Type: Array
```

Reply:
maybe?

```suggestion
* Type: Array | String
```

Reply:
Yeah sure that would work, I just redirect your attention back to every OTHER array definition in our docs that don't have this.

Reply:
good point 👍 I think it's better to be consistent